openstates/me/bills.py

```diff
@@ -249,37 +249,84 @@ def scrape_bill(self, bill, chamber):
                     vdoc.make_links_absolute(ver_url)
 
                     # various versions: billtexts, billdocs, billpdfs
-                    bill_text_html = vdoc.xpath(
-                        'string(//a[contains(@href, \
-                        "billtexts/")][1]/@href)'
-                    )
-                    bill_text_rtf = vdoc.xpath(
-                        'string(//a[contains(@href, \
-                        "billdocs/")][1]/@href)'
-                    )
-                    bill_text_pdf = vdoc.xpath(
-                        'string(//a[contains(@href, \
-                        "getPDF.asp")][1]/@href)'
-                    )
-
-                    if bill_text_html:
-                        bill.add_version_link(
-                            "Initial Version", bill_text_html, media_type="text/html"
+                    v_links = []
+                    for v in range(
```

Comment:
would it make more sense to iterate over the result of the vdoc.xpath(...story_heading) instead of using this?

Reply:
I tried to iterate over the result of story_heading but kept running into xpath issues where it kept picking up all of the version types urls during each loop through (eg. all the pdf urls were returned up at once).